scss/civicrm/search/pages/_search-builder.scss

```diff
@@ -0,0 +1,32 @@
+.CRM_Contact_Form_Search_Builder {
+  #map-field tr td {
+    padding: 10px 20px 4px !important;
+  }
+
+  h3 {
+    @include section-title;
+  }
+
+  .crm-search-builder-add-row .form-item,
+  .crm-search-builder-add-block {
+    text-decoration: none;
+
+    .submit-link {
+      color: $brand-primary;
+      margin: 0;
+    }
+
+    &:before {
+      @include fa-icon($font-size-base, $fa-var-plus);
+      color: $brand-primary;
+    }
+  }
+
+  #map-field .crm-search-builder-add-row td.form-item {
+    padding-bottom: 15px !important;
+  }
+
+  #crm-submit-buttons {
+    padding: 25px 20px;
```

Comment:
this looks a very generic styling ("apply paddings to the submit buttons"), is this really necessary? don't we have this style already somewhere else?